apps/base-docs/tutorials/docs/0_run-a-base-node.md

```diff
@@ -100,33 +100,47 @@ Syncing your node may take **days** and will consume a vast amount of your reque
 
 If you're a prospective or current Base Node operator and would like to restore from a snapshot to save time on the initial sync, it's possible to always get the latest available snapshot of the Base chain on mainnet and/or testnet by using the following CLI commands. The snapshots are updated every week.
 
-#### Full
-**Mainnet**
+#### Restoring from snapshot
+
+In the home directory of your Base Node, create a folder named `geth-data`. If you already have this folder, remove it to clear the existing state and then recreate it. Next, run the following code and wait for the operation to complete.
+
+**Testnet (Full)**
 
 ```
-wget https://mainnet-full-snapshots.base.org/$(curl https://mainnet-full-snapshots.base.org/latest)
+wget https://sepolia-full-snapshots.base.org/$(curl https://sepolia-full-snapshots.base.org/latest)
 ```
 
-**Testnet (Sepolia)**
+**Testnet (Archive)**
 
 ```
-wget https://sepolia-full-snapshots.base.org/$(curl https://sepolia-full-snapshots.base.org/latest)
+wget https://base-sepolia-archive-snapshots.s3.us-east-1.amazonaws.com/$(curl https://base-sepolia-archive-snapshots.s3.us-east-1.amazonaws.com/latest)
```

Comment:
I just tried this and it didn't work for me

Reply:
Fixed: [47d0a7a](https://github.com/base-org/web/pull/457/commits/47d0a7a5a25d679aefa5c186acd069291cdcb15b)